pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BrokerRequestHandler.java

```diff
@@ -21,11 +21,15 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.ThreadSafe;
+import org.apache.pinot.annotations.InterfaceAudience;
+import org.apache.pinot.annotations.InterfaceStability;
 import org.apache.pinot.broker.api.RequestStatistics;
 import org.apache.pinot.broker.api.RequesterIdentity;
 import org.apache.pinot.common.response.BrokerResponse;
 
 
+@InterfaceAudience.Public
+@InterfaceStability.Stable
```

Comment:
Let's not annotate this class for now. I don't know if start() and shutDown() should be in the interface

Reply:
I can mark it evolving - but it is something that is being depended on outside of the project, so we should atleast make that aspect clear.

Reply:
I don't know if that is true because so far we don't allow user to plug in their own request handler.

Reply:
Right, its not pluggable currently. I will remove the annotation.